package.json

```diff
@@ -86,16 +88,17 @@
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-mocha": "^4.11.0",
     "eslint-plugin-react": "^7.1.0",
-    "hock": "^1.2.0",
-    "mocha": "^2.2.5",
-    "npm": "~2.14.5",
+    "mocha": "^5.2.0",
+    "nock": "^10.0.6",
+    "npm": "^6.5.0",
```

Comment:
Where do we use this as a programmatic dependency? Shouldn't this be filled in by a `Dockerfile`, not as a dependency?

Reply:
@SivanMehta we spawn npm for testing certain endpoints for dist-tag and the like in the tests. it should be used in the `test/macros.js`

Reply:
Does `spawn('npm')` require `npm` as a dependency or does require the command to be present in the CI environment? The usage you described is [here](https://github.com/godaddy/warehouse.ai/blob/master/test/macros.js#L197).

Reply:
@SivanMehta ah you are right, it was never needed and there may be some more dependencies that can be removed. Thanks